src/ucp/core/ucp_context.c

```diff
@@ -2106,3 +2145,14 @@ void ucp_context_get_mem_access_tls(ucp_context_h context,
         }
     }
 }
+
+UCS_F_CTOR void ucp_global_init(void)
+{
+    UCS_CONFIG_ADD_TABLE(ucp_config_table, &ucs_config_global_list);
+    UCP_AM_HANDLER_FOREACH(UCP_AM_HANDLER_ENTRY);
```

Comment:
@yosefe is it ok ti initialise ```ucp_am_handlers``` such way? alternative we may change ucp_am_handlers type to  ```ucp_am_handler_t* ucp_am_handlers[]``` and initialise it statically (but it could affect to fast path)

Reply:
yes it is good
do we need to add ucp_global_init to undefined symbols in pkg-config?

Reply:
> do we need to add ucp_global_init to undefined symbols in pkg-config?

will add in https://github.com/openucx/ucx/pull/7842